package.json

```diff
@@ -1,13 +1,13 @@
 {
     "name": "everforest-moist",
     "displayName": "everforest-moist",
-    "description": "This theme is a blended version of the original everfoest themes.",
+    "description": "This theme is a blended version of the original everforest themes.",
     "version": "0.0.2",
     "LICENSE": "ISC",
     "publisher": "MrMartian",
     "repository": {
       "type": "git",
-      "url": "https://github.com/CraigglesO/everforest-moist-vscode"
```

Comment:
Thanks for the effort. however, this doesn't link to anything anymore. https://github.com/CraigglesO/everforest-moist-vscode doesn't exist.

Reply:
@CraigglesO That's why I have changed the code

Reply:
Right, when I copy paste the URL you have changed to: https://github.com/CraigglesO/everforest-moist-vscode
I get a 404. The original which is https://github.com/CraigglesO/everforest-moist works however. Can you change it back? Click the links to see what I mean

Reply:
Wow, Sorry I got that mixed up -_- thank you so much for the clarification.